debian/patches/series

```diff
@@ -0,0 +1 @@
+001_dont_scan_app_root.patch
```

Comment:
It's mad idea to provide a patch in the repository where we have original sources!!! :D

But even if we go ok with adding a patch, I strictly consider it as a separate pull request.

Reply:
Problem is that if flags |= CLocatorAPI::flScanAppRoot; is present binary scans directory where it's placed, and in case of debian package it's /usr/bin
And in this case there is:
ls /usr/bin/|wc -l
2714 files
It takes a lot of time to do so.
I would prefer to make it a cmake switch rather than patch, but as it's now it's broken.
I think that patch as temporary solution and replacing it with cmake switch in future is ok.